gapic/src/main/com/google/gapid/util/Colors.java

```diff
@@ -42,6 +44,18 @@ public static RGB rgb(double r, double g, double b) {
     return new RGB((int)(r * 255), (int)(g * 255), (int)(b * 255));
   }
 
+  public static RGB getRandomColor(int index) {
+    float hue = (float)Math.toDegrees(index * GOLDEN_ANGLE) % 360;
+    return new RGB(hue, 1f, 1f);
```

Comment:
Made a small adjustment to the color generator.

The only justification is that it's simpler and easier to understand. What does normalizing a color mean (especially without weighting the channels differently)?

https://screenshot.googleplex.com/oc8HLpgFAre - As a comparison of the first 10 colors generated by each algorithm.